tests/integration/targets/xfs_quota/tasks/pquota.yml

```
@@ -118,7 +118,7 @@
   - name: Assert project limits results for xft_quotaval after re-apply
     assert:
       that:
-      - not test_pquota_project_after.changed
+      - test_pquota_project_after.changed
```

This should nowadays be written as:
```suggestion
      - test_pquota_project_after is changed
```
(With `is not` if `changed` should be `false`.)

---

Good to know, thanks